tests/integration/simple_infillers/test_decompose_collection_with_time_dep_ratio.py

```diff
@@ -172,7 +173,7 @@ def test_db_error_preexisting_follow_data(self, test_db):
             tcruncher.infill_components(variable_follower, variable_leaders, test_db)
 
     def test_construct_consistent_error_multiple_units(self, test_db):
-        # test that crunching fails if there's no data about the follower gas in the
+        # test that crunching fails if there's no bad units in the
```

Comment:
This line of description I do not understand

Reply:
ok, reworded